### ISSUE TITLE
Add a list of running processes to the admin backend

### ISSUE THREAD

**Issue by nuest** [52North/WPS#208]

**Comment by jiwalker-usgs** [comment]
@nuest We've implemented this on our end and have been meaning to contribute it back.  You can check out http://cida.usgs.gov/gdp/process/dashboard and the entry point in code https://github.com/USGS-CIDA/geo-data-portal/blob/master/gdp-process-wps/src/main/java/gov/usgs/cida/gdp/wps/service/ProcessListService.java.  The big change that made it tricky to contribute is a reshuffling of the database schema.  Let me know if you'd like to meet sometime to go over our changes.

**Comment by nuest** [comment]
Thanks for getting in touch! Just for my understanding: So you're using the PostgreSQL database to store the results, and also create an entry in the database for every running process?

**Comment by jiwalker-usgs** [comment]
Yes, we split the PostgreSQL implementation up into several tables.  The request table tracks the incoming requests, and the response table holds the current status of the process and eventually the pointer to the outputs.  We haven't yet bitten off the process management (stopping running processes, etc).  We've set up a liquibase changelog to manage the schema and any changes that might need to be made going forward.
